deployment/go.mod

```diff
@@ -10,6 +10,7 @@ replace github.com/smartcontractkit/chainlink/v2 => ../
 require github.com/smartcontractkit/chainlink/v2 v2.0.0-20250128231431-9279badae2f0
 
 require (
+	github.com/AlekSi/pointer v1.1.0
```

Comment:
:( Feels a bit left-pady

Reply:
Also if you wanted to add this you'd be missing a go.sum update I would have thought